docs/reference/src/language/syntax/types.md

```diff
@@ -103,11 +103,15 @@ All colors and brushes define the following methods:
     Returns a new color with the opacity decreased by `factor`.
     The transparency is obtained by multiplying the alpha channel by `(1 - factor)`.
 
-
 -  **`with_alpha(alpha: float) -> brush`**
 
     Returns a new color with the alpha value set to `alpha` (between 0 and 1)
 
+- **`to-hsv()->{hue: float, saturation: float, value: float}`**
+
+    Returns a struct with the fields in the HSV colorspace.
+    Hue is between 0 and 360 while saturation and value are between 0 and 1.
+
```

Comment:
Either this:

```suggestion
- **`to-hsv()->{hue: float, saturation: float, value: float, alpha: float}`**

    Converts this color to the HSV color space and returns a struct with the `hue`, `saturation`, `value`,
    and `alpha` fields. `hue` is between 0 and 360 while `saturation`, `value`, and `alpha` are between 0 and 1.

```

or

```suggestion
- **`to-hsv()->{hue: float, saturation: float, value: float}`**

    Converts this color to the HSV color space and returns a struct with the `hue`, `saturation`, and `value`
    fields. `hue` is between 0 and 360 while `saturation` and `value` are between 0 and 1.
    The Color's alpha value is discarded.
```

I think the former makes more sense though, as we ... don't really discard it.